src/index.js

```diff
@@ -6,6 +6,7 @@ var Requestor = require('./Requestor');
 var Identity = require('./Identity');
 var utils = require('./utils');
 var messages = require('./messages');
+var store = require('./store');
```

Comment:
Does it matter than we require `store.js` here while the module is actually named `Store.js` (capital `S`)?

Reply:
Good catch. That may cause problems in some environments. Fixed!